core/src/App.svelte

```diff
@@ -2623,6 +2790,16 @@
         left: calc(#{$leftNavWidthCollapsed} + #{$globalNavWidth});
       }
     }
+    :global(.lui-semiCollapsible:not(.semiCollapsed)) {
```

Comment:
Not sure if it still matters at this point - but I just saw this. On our side, we tried to avoid top-level global styles because they are not scoped and we had issues with styling conflicts.

You have to use top-level `:global` here because the class `lui-semiCollapsible` is set in code in this component. If there is a way to change that to a [`class:name` directive](https://svelte.dev/docs/element-directives#class-name) in the template you might be able to change this to something like `.lui-semiCollapsible:global(:not(.semiCollapsed))`